src/inmanta/main.py

```diff
@@ -182,7 +182,7 @@ def get_table(header: list[str], rows: list[list[str]], data_type: Optional[list
     return table.draw()
 
 
-@with_plugins(iter_entry_points("inmanta.cli_plugins"))
+@with_plugins(importlib_metadata.entry_points(group="inmanta.cli_plugins"))
```

Comment:
mypy is raising a warning here:
```
src/inmanta/main.py:185: error: Argument 1 to "with_plugins" has incompatible type "EntryPoints"; expected "Generator[EntryPoint, None, None]"  [arg-type]
```
I don't know how should I fix this

Reply:
```suggestion
@with_plugins(iter(importlib_metadata.entry_points(group="inmanta.cli_plugins")))
```

like so?

Reply:
[this doesn't seem to be maintained](https://github.com/click-contrib/click-plugins) I think it is good to figure out why it is here at all (I suspect LSM). 

We may want to vendor it or find a fork if we need it.

Reply:
https://github.com/click-contrib/click-plugins/issues/37

Reply:
yes, click-plugin is just outdated, perhaps we can vendor it?